client/app/scripts/liveblog-edit/views/settings.html

```diff
@@ -49,7 +49,7 @@ <h2 translate>General</h2>
                                 <div class="form-input">
                                     <select
                                       ng-model="settings.blogPreferences.theme"
-                                      ng-options="theme as theme.name for theme in settings.availableThemes track by theme"
+                                      ng-options="theme.name as theme.name + ' ' for theme in settings.availableThemes"
```

Comment:
The set theme is not shown as selected when you refresh the page or return to the settings page. The selected="selected" attribute is there on the correct option but it does not show right in both FF and Chrome. Very strange behavior. I have tried with another field that I created, something like theme.displayName but had the same issue. Having the select and the value expressions evaluate to something just a bit different seems to fix it.